VideosController.js

```diff
@@ -0,0 +1,44 @@
+
+var express = require('express');
+var router = express.Router();
+var Events = require('./Events');
+
+router.get('/', function (req, res) {
+    Events.find({
+        $or: [{event: "StreamCreated"}, {event: "StreamEnded"}]
+    }).sort({
+        createdAt: 1
+    }).exec(function (err, events) {
+        if (err) return res.status(500).send("There was a problem finding the video.")
+
+        videos = {};
+        for (let i in events) {
```

Comment:
@ericxtang is `events` an array? If so, using `of` rather than `in` makes more sense:`for (let i of events) { ... }`